PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3390,6 +3404,48 @@ bool FindReplaceDlg::replaceInOpenDocsConfirmCheck(void)
 	return confirmed;
 }
 
+void FindReplaceDlg::markedTextToClipboard(int indiStyle)

```

Comment:
According https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8943 ,  `markedTextToClipboard` method shouldn't belong to `FindReplaceDlg`

Reply:
Haha. Future issues affect current issues.

Reply:
> Haha. Future issues affect current issues.

Not really, markedTextToClipboard has been made more generic, so it should be in a place more generic :)

Reply:
Probably to do it well, we should cancel this PR, combine the two issues into one, and have a cohesive approach.
I'm usually in favor of separating things, but in this case maybe together is better.
If you agree, close this PR?

Reply:
Do you prefer to move markedTextToClipboard in the PR for #8943 ?

Reply:
Short answer: Yes. :-)